.github/CODEOWNERS

```diff
@@ -35,22 +30,22 @@
 /tests/modules/images/webp-uploads                                          @adamsilverstein @felixarntz @mitogh
 /tests/testdata/modules/images/webp-uploads                                 @adamsilverstein @felixarntz @mitogh
 
-# Module: WebP Support Health Check
-/modules/site-health/webp-support                                           @adamsilverstein @kirtangajjar @mitogh
-/tests/modules/site-health/webp-support                                     @adamsilverstein @kirtangajjar @mitogh
-/tests/testdata/modules/site-health/webp-support                            @adamsilverstein @kirtangajjar @mitogh
+# Module: WebP Support
+/modules/images/webp-support                                           @adamsilverstein @kirtangajjar @mitogh
+/tests/modules/images/webp-support                                     @adamsilverstein @kirtangajjar @mitogh
+/tests/testdata/modules/images/webp-support                            @adamsilverstein @kirtangajjar @mitogh
 
-# Module: Autoloaded Options Health Check
-/modules/site-health/audit-autoloaded-options                               @manuelRod
-/tests/modules/site-health/audit-autoloaded-options                         @manuelRod
-/tests/testdata/modules/site-health/audit-autoloaded-options                @manuelRod
+# Module: Audit Autoloaded Options
```

Comment:
See above, please keep the previous name of the module. It should only be moved to its new focus area directory, not be renamed.